pkg/workflow/executor/workflow_executor.go

```diff
@@ -61,50 +68,22 @@ func (e *WorkflowExecutor) StartWorkflow(startWorkflowRequest *model.StartWorkfl
 	return id, err
 }
 
-//MonitorExecution monitors the workflow execution
```

Comment:
why did we remove this?

Reply:
It wasn't removed, it's just above in the file: https://github.com/conductor-sdk/conductor-go/blob/f4cc31e89d764ea64a7750ab623a5acc12a2d832/pkg/workflow/executor/workflow_executor.go#L51-L56